packages/protocol/src/providers/polygon/WePiggyPolygon.sol

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.15;
+
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import {IVault} from "../../interfaces/IVault.sol";
+import {ILendingProvider} from "../../interfaces/ILendingProvider.sol";
+import {IAddrMapper} from "../../interfaces/IAddrMapper.sol";
+import {IComptroller} from "../../interfaces/compoundV2/IComptroller.sol";
+import {ICETH} from "../../interfaces/compoundV2/ICETH.sol";
+import {ICERC20} from "../../interfaces/compoundV2/ICERC20.sol";
+import {IGenCToken} from "../../interfaces/compoundV2/IGenCToken.sol";
+import {ICETH} from "../../interfaces/compoundV2/ICETH.sol";
+import {ICERC20} from "../../interfaces/compoundV2/ICERC20.sol";
+import {IWETH9} from "../../abstracts/WETH9.sol";
+import {SafeERC20} from "openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
+
+/**
+ * @title WePiggy Lending Provider.
+ * @author fujidao Labs
+ * @notice This contract allows interaction with WePiggy.
+ */
+
+//wmatic
```

Comment:
I suggest for code cleanliness to perhaps we remove these noted addresses.

Reply:
absolutely right! I missed it